state/service.proto

```diff
@@ -27,6 +27,10 @@ service Status {
     // To permit access, used server config options.
     // The request should be signed.
     rpc DumpVars(DumpVarsRequest) returns (DumpVarsResponse);
+    // ChangeState allows to change current node state of node.
+    // To permit access, used server config options.
+    // The request should be signed.
```

Comment:
I suggest leaving here only the call destination (what can be done)

Reply:
There’s description, that used in an other calls

Reply:
Oh, i see now.

Reply:
added TODO to decide later